test/functional/fixtures/tty-test.c

```diff
@@ -94,7 +96,11 @@ static void read_cb(uv_stream_t *stream, ssize_t cnt, const uv_buf_t *buf)
   uv_tty_init(&write_loop, &out, fileno(stdout), 0);
 
   uv_write_t req;
+#ifdef WIN32
+  uv_buf_t b = {.base = buf->base, .len = (ULONG)cnt};
```

Comment:
It is supposed to have spaces before closing and after opening figure braces, though this file is probably not linted.